Mage.Sets/src/mage/cards/r/RadhaHeartOfKeld.java

```diff
@@ -0,0 +1,69 @@
+package mage.cards.r;
+
+import mage.MageInt;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.condition.common.MyTurnCondition;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.decorator.ConditionalContinuousEffect;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.PermanentsOnBattlefieldCount;
+import mage.abilities.effects.common.continuous.BoostSourceEffect;
+import mage.abilities.effects.common.continuous.GainAbilitySourceEffect;
+import mage.abilities.effects.common.continuous.LookAtTopCardOfLibraryAnyTimeEffect;
+import mage.abilities.effects.common.continuous.PlayTheTopCardEffect;
+import mage.abilities.hint.common.MyTurnHint;
+import mage.abilities.keyword.FirstStrikeAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.filter.FilterCard;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterLandCard;
+
+import java.util.UUID;
+
+/**
+ * @author arcox
+ */
+public final class RadhaHeartOfKeld extends CardImpl {
+    private static final FilterCard filter = new FilterLandCard("play land cards");
+
+    public RadhaHeartOfKeld(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{R}{G}");
+
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.ELF);
+        this.subtype.add(SubType.WARRIOR);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // As long as it's your turn, Radha, Heart of Keld has first strike.
+        this.addAbility(new SimpleStaticAbility(new ConditionalContinuousEffect(
+                new GainAbilitySourceEffect(FirstStrikeAbility.getInstance(), Duration.WhileOnBattlefield),
+                MyTurnCondition.instance, "As long as it's your turn, {this} has first strike."
+        )).addHint(MyTurnHint.instance));
+
+        // You may look at the top card of your library any time, and you may play lands from the top of your library.
+        this.addAbility(new SimpleStaticAbility(new LookAtTopCardOfLibraryAnyTimeEffect()));
+        this.addAbility(new SimpleStaticAbility(new PlayTheTopCardEffect(filter)));
```

Comment:
You should add these effects to the same ability and modify the text so they're on the same line

Reply:
Fixed. Also changed her activated ability to say "Radha" instead of {this}.